staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/aggregator.go

```diff
@@ -192,7 +193,7 @@ func (s *specAggregator) buildOpenAPISpec() (specToReturn *spec.Swagger, err err
 	for _, specInfo := range specs {
 		// TODO: Make kube-openapi.MergeSpec(s) accept nil or empty spec as destination and just clone the spec in that case.
 		if specToReturn == nil {
-			specToReturn, err = aggregator.CloneSpec(specInfo.spec)
+			specToReturn, err = cloneSpec(specInfo.spec)
```

Comment:
the purpose of https://github.com/kubernetes/kube-openapi/pull/124 was to avoid the clone here